sustainml_cpp/include/sustainml_cpp/orchestrator/OrchestratorNode.hpp

```diff
@@ -199,7 +214,7 @@ class OrchestratorNode
 
     uint32_t domain_;
 
-    std::shared_ptr<OrchestratorNodeHandle> handler_;
+    OrchestratorNodeHandle* handler_;
```

Comment:
We may add a comment here stating that the deletion of the `handler_` is managed by the user. Also we may include it in the `destroy()` method and in the descriptiof of the `OrchestratorNodeHandle` inside a `@note`